spec/replication/replication_spec.rb

```diff
@@ -0,0 +1,166 @@
+require_relative "./replication_helper"
+
+describe "pglogical replication" do
+  let(:slave_db_name)  { "#{ActiveRecord::Base.connection.current_database}" }
+  let(:master_db_name) { "#{slave_db_name}_master" }
+  let(:rep_set_name)   { "test_rep_set" }
+  let(:sub_name)       { "test_subscription" }
+
+  let(:excluded_tables_regex) do
+    excluded_tables = @replication_config[:exclude_tables].join("|")
+    excluded_tables = "^(#{excluded_tables})$"
+    Regexp.new(excluded_tables)
+  end
+
+  let(:conn_info) do
+    config = VMDB::Config.new("database").config[:test]
```

Comment:
Prefer instead: `Rails.configuration.database_configuration["test"]`.  Note then, that all the keys are strings, so you'd have to change that below.